scripts/grafana-server/start-server

```diff
@@ -34,6 +34,17 @@ mkdir $PROV_DIR/dashboards
 cp ./scripts/grafana-server/custom.ini $RUNDIR/conf/custom.ini
 cp ./conf/defaults.ini $RUNDIR/conf/defaults.ini
 
+echo -e "Copying custom plugins from e2e tests"
```

Comment:
This is what allows e2e tests to load custom plugins required for the sandbox testing.